libs/components/layout/src/lib/modules/box/box-header.component.ts

```diff
@@ -6,5 +6,12 @@ import { Component } from '@angular/core';
 @Component({
   selector: 'sky-box-header',
   templateUrl: './box-header.component.html',
+  styles: [
+    `
+      ::ng-deep sky-help-inline {
```

Comment:
This will be unnecessary when you use the `sky-control-help-container` class.

Reply:
Removed this.